bindings/nodejs/index.d.ts

```diff
@@ -101,3 +119,15 @@ export class Lister {
 export class BlockingLister {
   next(): Entry | null
 }
+export class PresignedRequest {
+  /** Returns the HTTP method of this request. */
+  get method(): string
+  /** Returns the URI of this request. */
+  get uri(): string
+  /**
+   * Returns the headers of this request.
+   *
+   * The key of the map is the header name, and the value is the header value AS bytes.
+   */
+  headers(): Record<string, Array<number>>
```

Comment:
Is this normal in nodejs ecosystem?

Reply:
I'm not completely sure yet. This is still a draft.

According to the HTTP spec, the values should all be strings. I'm considering changing them to string.